include/fastdds_statistics_backend/StatisticsBackend.hpp

```diff
@@ -612,6 +623,14 @@ class StatisticsBackend
     FASTDDS_STATISTICS_BACKEND_DllAPI
     static std::string deserialize_guid(
             fastdds::statistics::detail::GUID_s data);
+
+    /**
+     * @brief Serialize entity guid to string format.
```

Comment:
Please correct the description

```suggestion
     * @brief Serialize entity guid from string.
```